gatsby-node.js

```diff
@@ -63,10 +167,8 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
       slug = slug.replace("/translations", "")
       const split = slug.split("/")
       split.splice(1, 1)
-      const originalPath = split.join("/")
-      if (outdatedMarkdownPages.includes(originalPath)) {
-        isOutdated = true
-      }
+
+      isOutdated = await checkIsMdxOutdated(node.fileAbsolutePath)
```

Comment:
Would it make sense to just work with relative paths? We already use this logic down below:
```
    const absolutePath = node.fileAbsolutePath
    const relativePathStart = absolutePath.indexOf("src/")
    const relativePath = absolutePath.substring(relativePathStart)
```

Reply:
Ah does `fs` need the absolute path? Probably 😄

Reply:
YEah makes the fs readfile easier.